hooks/useWallet.tsx

```diff
@@ -12,16 +14,31 @@ const INFURA_ID = process.env.NEXT_PUBLIC_INFURA_ID;
 
 const providerOptions = {
   walletconnect: {
-    package: WalletConnectProvider, // required
+    package: WalletConnectProvider,
     options: {
-      infuraId: INFURA_ID, // required
+      infuraId: INFURA_ID,
     },
   },
+  coinbasewallet: {
+    package: CoinbaseWalletSDK,
+    options: {
+      appName: 'CityDAO Parcel-0',
+      infuraId: INFURA_ID,
+      darkMode: true,
+    },
+  },
+  torus: {
```

Comment:
did these work for you? every time I tested them, they would throw errors.

Reply:
Looks like Torus works – not sure why 🤷 . However, Binance Chain doesn't work, and the console shows a silent error if you don't have it.